app/scripts/controllers/utils/utils.js

```diff
@@ -0,0 +1,34 @@
+///: BEGIN:ONLY_INCLUDE_IN(flask)
```

Comment:
`utils/utils.js` is not a really descriptive file

Reply:
Changed this to `app/scripts/flask/snaps-utilities.js`. Let me know if that's alright.